main.go

```diff
@@ -1263,12 +1392,17 @@ func generateChanges(auto bool, stage bool) {
 			}
 		}
 		if auto {
-			err = commitChanges(packageList)
+			err = commitChanges(packageList, false)
 			if err != nil {
 				logrus.Fatal(err)
 			}
 		}
 	}
+
+	if auto && icons {
```

Comment:
@adamkpickering 
Removed `override-icons` command option. 
If we call `auto` with `--icons` flag, this will be run.

Reply:
Yes this makes totally sense, modifying it.